apps/frontend/src/app/5_pages/AavePage/components/LendPositionsList/components/LendPositionAction/LendPositionAction.tsx

```diff
@@ -1,24 +1,41 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useState } from 'react';
 
 import { t } from 'i18next';
 
 import { Button, ButtonStyle } from '@sovryn/ui';
 
 import { translations } from '../../../../../../../locales/i18n';
 import { LendPosition } from '../../LendPositionsList.types';
+import { WithdrawModalContainer } from '../WithdrawModal/WithdrawModalContainer';
 
 type LendPositionActionProps = {
   position: LendPosition;
 };
 
 export const LendPositionAction: FC<LendPositionActionProps> = () => {
+  const [isWithdrawModalOpen, setIsWithdrawModalOpen] =
+    useState<boolean>(false);
+
+  const handleWithdrawClick = useCallback(() => {
+    setIsWithdrawModalOpen(true);
+  }, []);
+
+  const handleWithdrawClose = useCallback(() => {
+    setIsWithdrawModalOpen(false);
+  }, []);
+
```

Comment:
"Unnecessary use of useCallback: When a function is quite simple, optimizing it with useCallback can be less performant than simply recreating the function multiple times."

```suggestion
  const handleWithdrawClick = () => {
    setIsWithdrawModalOpen(true);
  };

  const handleWithdrawClose = () => {
    setIsWithdrawModalOpen(false);
  };

```

Reply:
Totally!! Commited! thanks!